data/types/blood/glucose/glucose.go

```diff
@@ -33,3 +36,75 @@ func (g *Glucose) Normalize(normalizer data.Normalizer) {
 		g.Value = dataBloodGlucose.NormalizeValueForUnits(g.Value, units)
 	}
 }
+
+func roundToEvenWithDecimalPlaces(v float64, decimals int) float64 {
+	if decimals < 1 {
```

Comment:
Consider just allowing 0 and negative numbers? The math still works out just fine.